src/Card.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './card.css';
+
+function Card (props){ 
+    const [ email, setEmail ] = React.useState('Loading...');
+    const [ firstName, setFirstName ] = React.useState('Loading...');
+    const [ lastName, setLastName ] = React.useState('Loading...');
+    const [ avatar, setAvatar ] = React.useState('Loading...');
+    
+    React.useEffect(() => {
```

Comment:
Totally up to you, but I tend to prefer to import my hooks in the import statement so that I don't have to type `React.` over and over again. Some people prefer to use the functions off of the default export though, so your call!